### ISSUE TITLE
Move global UA grid-to-obs verification over from the global workflow

### ISSUE THREAD

**Issue by JeffHamiltonNOAA** [NOAA-GSL/VxLegacyIngest#428]
Problem

- Global grid-to-obs verification (ACC, UA, and surface) is unstable, with repeated failures even after major adjustments to the workflow and testing. 

Issues

- The verification is unique as it resides in the global rocoto workflow framework, quite a different paradigm than our regional verification suite.

- Being in the global workflow means that it resides in a gray area as to who is responsible for managing it when things break

Changes in the global workflow has caused numerous interruptions and code adjustments over the past 6 months

Examples:

- Feb 2021: directory structure changes caused verification to fail. Verification scripts were completely overhauled inside the global workflow to be completely detached from any hard coded directory structure, BUT this required the governing XML file to be properly configured in order to pass the correct file

- March 2021: Default output switches from half-degree to quarter degree, necessitating code changes in executables that were written for the global workflow with half-degree hard-coded (this is unique to the global workflow, our normal verification does not care what resolution it is) and in ACC verification suite using MET

- April 2021: Changes to the UA code to mimic what was done with surface due to continued directory and filename changes, causing continued outages

May 2021: ACC verification fails (outstanding issue)

Solution

- Remove all verification from the global workflow and simply add it to our “regional” framework (not designed to be regional, it is just how it worked out in practice)

- Verification group will be responsible for any verification outages, with a clear break in responsibility being the point at which model output is generated

**Comment by JeffHamiltonNOAA** [comment]
I've found that when processing the upper-air variables of the GSL global models I had to scan the files in a different way in order to grab the correct grid points (what I've been fighting all this time). Typically, we scan files from west to east and south to north (operational GFS, HRRR, etc.), but when running tests with rt_gfsv16_gf I found that I had to scan north to south instead. What made this stranger was that when running surface verification, I scanned the typical way and got the correct grid points without issue, so clearly something is inconsistent somewhere in output generation. I've informed Judy, but have code to work around this.

**Comment by JeffHamiltonNOAA** [comment]
Surface verification and UA verification are running for the following realtime global runs:

rt_gfsv16_gf
rt_gfsv16_mynn
rt_gfsv16_thmp_subcyc

A few global runs (CCPP-C384 and CCPP-Chem) differs in the number of levels between variables as I progress through the file. As an example, U and V components of wind do not have a 40 mb and 15 mb level, however TMP, HGT, and RH do. I've emailed Judy about this and she is looking into it, but this prevents proper verification at this time. Will add these two as soon as I have an update from her.

**Comment by JeffHamiltonNOAA** [comment]
Judy has talked with global folks and they are investigating the post-processing

**Comment by JeffHamiltonNOAA** [comment]
The global models I was waiting on are no longer relevant from a verification perspective. This issue can now be closed
